ddp-workspace/projects/ddp-testboston/src/app/models/ddpError.ts

```diff
@@ -0,0 +1,9 @@
+import { ErrorType } from './errorType';
+
+export class DdpError extends Error {
+    errorType: ErrorType;
+    constructor(msg: string, errorType: ErrorType) {
+        super(msg);
+        this.errorType = errorType;
+    }
+}
```

Comment:
Surprised we have not had one of these before. Well here it is. I am using with new component.

Reply:
Cool!